### ISSUE TITLE
No parameters are returned

### ISSUE THREAD

**Issue by JasperRev** [aws/aws-dotnet-extensions-configuration#53]
I am not able to retrive any parameters.  This is an aspnet core 2 program.  I've added and verified my credentials are correct in my launchsettings (environment vars).

In program cs ...
`
        public static IWebHostBuilder CreateWebHostBuilder(string[] args) =>
            WebHost.CreateDefaultBuilder(args)
        .ConfigureAppConfiguration(builder =>
        {

            var awsOptions = new AWSOptions();
            awsOptions.Credentials = new EnvironmentVariablesAWSCredentials();
             builder.AddJsonFile("appsettings.json");
            builder.AddSystemsManager("/config", awsOptions, false, TimeSpan.FromMinutes(5));
        }).UseStartup<Startup>();
    }`

In Startup.cs

`
        public void ConfigureServices(IServiceCollection services)
        {
            services.AddMvc().SetCompatibilityVersion(CompatibilityVersion.Version_2_2);
            var awsOptions =new AWSOptions();
            awsOptions.Credentials = new EnvironmentVariablesAWSCredentials();
            services.AddDefaultAWSOptions(awsOptions);
        
            var myconfig = Configuration.GetSection("test");
         }`

myconfig comes back with a value of null - my parameter is /config/test/something.

The bottom line is that i want to pass in my environment vars, and i know it works because when i dont i get an error that i dont have access.  Why am i getting null essentially?  It doesnt look to be pulling from the parameter store.  Thank you!

**Comment by JasperRev** [comment]
User error.

**Comment by safv12** [comment]
@JasperRev what was the user error? I'm struggling with the same case.

**Comment by VaclavElias** [comment]
Just a reference for anyone, make sure that a location you created in your AWS parameters is same as the one you are referencing in your code.

![image](https://user-images.githubusercontent.com/4528464/107263077-39c7af00-6a39-11eb-9218-0715a27cc2df.png)
